Source/Text/SvgFontDefn.cs

```diff
@@ -54,7 +54,7 @@ public SizeF MeasureString(ISvgRenderer renderer, string text)
         {
             var result = new List<RectangleF>();
             using (var path = GetPath(renderer, text, result, true)) { }
-            var nonEmpty = result.Where(r => r != RectangleF.Empty);
+            var nonEmpty = result.Where(r => r != RectangleF.Empty).ToArray();
```

Comment:
Prevent multiple enumeration

Reply:
I think it could be rewritten as

 ```
float? firstLeft = null;
 float? lastRight = null;
 foreach (var rect in result.Where(r => r != RectangleF.Empty))
 {
     firstLeft ??= rect.Left;
     lastRight = rect.Right;
 }       

 if (firstLeft == null) return SizeF.Empty;
 return new SizeF(lastRight.Value - firstLeft.Value, Ascent(renderer));
```

So only one iteration is done instead of 3 Itterations.

Reply:
@inforithmics Thanks - implemented your change.